start_positions:
  arm1:
    orientation:
    - 0.8941549475460064
    - 0.40673623987374274
    - -0.1803229259846407
    - 0.05036073189207779
    position:
    - 0.9830024927504447
    - 0.15855712266597
    - 1.1462362212321988
  arm2:
    orientation:
    - -0.7891802708626706
    - -0.5836044272209543
    - 0.03285427715691526
    - 0.18847007476445868
    position:
    - 0.9482869312931976
    - -0.16625569408640606
    - 1.1702596272219692
  diana7:
    orientation:
    # - -0.6155382697445587
    # - 0.6308624728179258
    # - -0.33861431452859436
    # - 0.32934104643402223
    - 0.5
    - -0.5
    - 0.5
    - -0.5
    position:
    # - 0.867056037696641
    # - 0.019312428838694235
    # - 0.7568464799371728
    - 0.867056037696641
    - 0.019312428838694235
    - 0.8568464799371728
